modules/node_modules/@ncigdc/components/Home.js

```diff
@@ -145,21 +145,28 @@ const Home = compose(
 
       setState(state => ({ ...state, humanBodyLoading: !this.props.humanBodyData.length }));
 
+      const data = transformChartData({ hits: this.props.humanBodyData })
+        .sort((a, b) => a._key > b._key ? 1 : -1);
+
       if (this.props.humanBodyData.length) {
         setTimeout(() => {
           const root = document.getElementById('human-body-root');
           sapien.default({
             clickHandler: d => {
-              const query = {
-                filters: JSURL.stringify(makeFilter([
-                  {
-                    field: 'primary_site',
-                    value: [d._key.replace(/-/g, ' ')],
-                  }],
-                  false
-                )),
-              };
-              push({ pathname: '/projects', query });
+              const key = d._key.replace(/-/g, ' ');
+
+              if (data.find(x => x._key === key)) {
```

Comment:
We probably shouldn't have a pointer when clicking won't do anything. not sure if that makes sense here or if it would need to be in lolliplot? 
@cy did it here by putting the onclick in the data https://github.com/NCI-GDC/portal-ui/blob/relay/modules/node_modules/%40ncigdc/components/Charts/BarChart.js#L108